Source/carthage/CopyFrameworks.swift

```diff
@@ -47,7 +47,7 @@ public struct CopyFrameworksCommand: CommandType {
 private func codeSigningIdentity() -> SignalProducer<String?, CarthageError> {
 	return SignalProducer.try {
 		if codeSigningAllowed() {
-			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0 }
+			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0.isEmpty ? nil : $0 }
```

Comment:
Calling `codesign()` will be skipped on following: https://github.com/Carthage/Carthage/blob/master/Source%2FCarthageKit%2FXcode.swift#L1232
by `codesigningIdentity` is `nil`